src/main/java/io/github/jhipster/online/util/SanitizeInputs.java

```diff
@@ -29,4 +29,16 @@ public static String sanitizeInput(String inputString) {
             return null;
         return inputString.replaceAll("[\n\r\t]", "_");
     }
+
+    public static boolean isAlphaNumeric(String inputString) {
+        if (inputString == null)
+            return false;
+        return inputString.matches("[a-zA-Z0-9]*");
```

Comment:
The pattern should be compiled only once (statically)  not for each call to the method.

Reply:
Thanks. Done 👍🏼